src/index.html

```diff
@@ -7,8 +7,55 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
```

Comment:
Please, follow the [checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md), 13 in this case: 
![image](https://github.com/mate-academy/layout_moyo-header/assets/86923089/3f3a5bf9-8dd5-4a5c-8bb9-876c435693d7)

Make sure **all** elements is written as in the good example